extensions/ql-vscode/src/vscode-tests/cli-integration/run-remote-query.test.ts

```diff
@@ -180,6 +238,41 @@ describe('Remote queries', function() {
     }
   });
 
+  function verifyQlPack(qlpackPath: string, queryPath: string, packName: string, packVersion: string, pathSerializationBroken: boolean) {
+    const qlPack = yaml.safeLoad(fs.readFileSync(qlpackPath, 'utf8')) as QlPack;
+
+    if (pathSerializationBroken) {
+      // the path serialization is broken, so we force it to be the path in the pack to be same as the query path
+      qlPack.defaultSuite![1].query = queryPath;
+    }
```

Comment:
```suggestion
    if (pathSerializationBroken) {
      // the path serialization is broken, so we force the path in the pack to be same as the query path
      qlPack.defaultSuite![1].query = queryPath;
    }
```

or something like this?

Reply:
Sure...I think that was laziness from me on relying on co-pilot too much.:)